app/controllers/organizations/users_controller.rb

```diff
@@ -1,11 +1,18 @@
 module Organizations
   class UsersController < ApplicationController
+    before_action :authorize_user, only: %i[index]
```

Comment:
I didn't run code 🤔 But can we use just `authorize!` here?
```suggestion
    before_action :authorize!, only: %i[index]
```